frontend/src/DiscoverMovie/Table/AddListMovieRow.js

```diff
@@ -121,6 +127,24 @@ class AddListMovieRow extends Component {
                   <Link
                     {...linkProps}
                   >
+                    <span style={{ marginRight: `${linkMenuMargin}em` }} className={styles.links}>
+                      <Popover
+                        anchor={
+                          <Icon
+                            name={icons.EXTERNAL_LINK}
+                            size={15}
+                          />
+                        }
+                        title="Links"
+                        body={
+                          <MovieDetailsLinks
+                            tmdbId={tmdbId}
+                            imdbId={imdbId}
+                            youTubeTrailerId={youTubeTrailerId}
+                          />
+                        }
+                      />
+                    </span>
```

Comment:
Same note as overviews, would this be better off in the actions column beside the exclusions button for consistency?